ntoskrnl/ex/sysinfo.c

```diff
@@ -718,17 +718,15 @@ QSI_DEF(SystemPerformanceInformation)
      *   Add up all the used "Committed" memory + pagefile.
      *   Not sure this is right. 8^\
      */
-    Spi->CommittedPages = MiMemoryConsumers[MC_SYSTEM].PagesUsed +
-                          MiMemoryConsumers[MC_USER].PagesUsed +
-                          MiUsedSwapPages;
+    Spi->CommittedPages = MmTotalCommittedPages;
```

Comment:
Calculation is moved, so should its related comment.

Reply:
Yes